<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1053</width>
    <height>844</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Exp">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>100</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Exp</string>
    </property>
   </widget>
   <widget class="QLabel" name="Lv">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>150</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Lv</string>
    </property>
   </widget>
   <widget class="QLabel" name="health">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>210</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>health</string>
    </property>
   </widget>
   <widget class="QLabel" name="damage">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>280</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>攻击力</string>
    </property>
   </widget>
   <widget class="QLabel" name="speed">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>350</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>速度</string>
    </property>
   </widget>
   <widget class="QLabel" name="timeleft">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>151</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="MapView" name="myview">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>711</width>
      <height>671</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
   </widget>
   <widget class="QPushButton" name="h_enhance">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>470</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>生命</string>
    </property>
   </widget>
   <widget class="QPushButton" name="a_enhance">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>470</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>攻击力</string>
    </property>
   </widget>
   <widget class="QPushButton" name="s_enhance">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>470</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>速度</string>
    </property>
   </widget>
   <widget class="QLabel" name="enhance_label">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>400</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;华文细黑&quot;;</string>
    </property>
    <property name="text">
     <string>强化系统（暂无强化）</string>
    </property>
   </widget>
   <widget class="QLabel" name="wave">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>40</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>当前波次：1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="af_enhance">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>570</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>攻击频率</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ar_enhance">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>570</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>射程</string>
    </property>
   </widget>
   <widget class="QPushButton" name="as_enhance">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>570</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>子弹速度</string>
    </property>
   </widget>
   <widget class="QLabel" name="money">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>30</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>当前金币：</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1053</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MapView</class>
   <extends>QGraphicsView</extends>
   <header location="global">mapview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
